clients/src/main/java/org/apache/kafka/common/requests/TxnOffsetCommitResponse.java

```diff
@@ -91,7 +91,9 @@ public int throttleTimeMs() {
 
     @Override
     public Map<Errors, Integer> errorCounts() {
-        return errorCounts(errors().values());
+        return errorCounts(data.topics().stream()
+                .flatMap(topic -> topic.partitions().stream())
```

Comment:
Any reason why we don't do the map inside the flatMap?

Reply:
As above.